models/app.model.js

```diff
@@ -90,3 +90,14 @@ exports.patchVotes = (article_id, inc_votes) => {
       return rows[0];
     });
 };
+
+exports.removeComment = (comment_id) => {
+  return db.query(
+    `
+    DELETE FROM comments
+    WHERE comment_id = $1
+    RETURNING *;
+    `,
+    [comment_id]
+  );
```

Comment:
This is a promise so you can and should add a .then(...) here to do any error handling if needed